/**:
  ros__parameters:
    can_interface: "can0"
    can_baudrate: 500000
    vesc_ids: [49, 124, 81, 94]
    wheel_labels: ["front_left", "front_right", "rear_left", "rear_right"]
    wheel_radii: [0.115, 0.115, 0.115, 0.115]       # радиус в метрах
    wheel_poles: [30, 30, 30, 30]                  # количество полюсов (магнитов)
    wheel_abs_min_erpm: [900, 900, 900, 900]       # минимальный ERPM для движения
    wheel_base: 0.3                                # расстояние между передней и задней осью (м)
    publish_rate: 50.0
    speed_max: 23250.0
    speed_min: -23250.0